### PR TITLE
Fix crashes of MPS backend

### PR DESCRIPTION
**Pull request by sunlin-link**

### CODE REVIEW

**Comment by sunlin-link** [comment]
Fix: https://github.com/intel/webml-polyfill/issues/171

File: services/ml/compilation_impl_mac.mm [hunk 1]
```diff
@@ -1082,15 +1087,15 @@ void ComputeBNNSOffsetForImplicitPadding(
 
     auto impl =
         std::make_unique<ExecutionImplMac>(this, std::move(memory_handle));
-    if (!impl->IsValid()) {
-      std::move(callback).Run(mojom::BAD_DATA, nullptr);
-      return;
-    }
+    bool result = impl->IsValid();
```
Comment (fujunwei) [inline services/ml/compilation_impl_mac.mm]: Could you please explain why not return nullptr?

Comment (fujunwei) [reply services/ml/compilation_impl_mac.mm]: The mojom interface definition  need an optional type with ? symbol.
https://github.com/otcshare/chromium-src/blob/webml/services/ml/public/interfaces/compilation.mojom#L25

CreateExecution()
    => (int32 resultCode, ExecutionInitParams? init_params);

https://chromium.googlesource.com/chromium/src/+/master/mojo/public/tools/bindings/README.md

**Comment by fujunwei** [comment]
thanks, LGTM.

**Pull request merged**